observer/observer.ts

```diff
@@ -47,10 +47,9 @@ namespace ObserverPattern {
         private state: number;
         private subject: ConcreteSubject;
 
-        constructor (subject: ConcreteSubject, name: string) {
+        constructor ( name: string) {
```

Comment:
Did you mean to leave a space before the param? Just curious 😉

Reply:
> Did you mean to leave a space before the param? Just curious
Be cool, now.